tests/Plugins/GitHubEnterprise_EndToEndTest.py

```diff
@@ -0,0 +1,377 @@
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequireConversationResolution(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequireConversationResolution-false"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequireLinearHistory(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequireLinearHistory-true"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequirePullRequests(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequirePullRequests-false"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequireSignedCommits(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequireSignedCommits-false"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequireStatusChecksToPass(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequireStatusChecksToPass-false"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+    # ----------------------------------------------------------------------
+    def test_RequireUpToDateBranches(self, pat_args, snapshot):
+        result = CliRunner().invoke(
+            app,
+            pat_args + ["--GitHub-RequireUpToDateBranches-false"],
+        )
+
+        assert result.exit_code == -1, result.output
+        assert ScrubDurationGithuburlAndSpaces(result.stdout) == snapshot
+
+
+# ----------------------------------------------------------------------
+# ----------------------------------------------------------------------
+# ----------------------------------------------------------------------
+@pytest.fixture
+def args() -> list[str]:
+    return [
+        "--include",
+        "GitHub",
+        "--GitHub-url",
+        "https://github.gatech.edu/vagrawal38/RepoAuditor",
```

Comment:
Should have an sse repo instead of a personal repo?

Reply:
This is tricky since then we need devs to have a PAT from the SSE organization which isn't scalable. This is the same issue we had before with the https://github.com/gt-sse-center/RepoAuditor website. 🙁

Reply:
@mtanneau I managed to figure this out.